spec/support/landing_folder_support/file_objects/et3_pdf_file/header_section.rb

```diff
@@ -0,0 +1,27 @@
+require_relative './base'
+module EtApi
+  module Test
+    module FileObjects
+      module Et3PdfFileSection
+        class HeaderSection < ::EtApi::Test::FileObjects::Et3PdfFileSection::Base
+          def has_contents_for?(response:) # rubocop:disable Naming/PredicateName
+            # @TODO Review this conditional after march 2019 - once the pdf has been sorted, it should have the date_received and rtf fields
```

Comment:
There are a couple of TODO's in this PR as the ET3 template is missing a few fields - so I have got it working with the current one, then we will get the pdf template improved to make it consistent.

Reply:
A wise man once said to me that all TODOs should be tied to a ticket number :grin: